packages/graphql-transformer-core/src/util/transformConfig.ts

```diff
@@ -50,6 +50,12 @@ export interface TransformConfig {
      * generalizes this behavior.
      */
     Migration?: TransformMigrationConfig;
+
+    /**
```

Comment:
Change the comment since the flag has been changed to version number

Reply:
Changed comment